crates/bifrost/src/providers/replicated_loglet/network.rs

```diff
@@ -106,7 +106,7 @@ impl<T: TransportConnect> SequencerDataRpcHandler<T> {
 
         let global_tail = loglet.known_global_tail();
 
-        let loglet_commit = match loglet.enqueue_batch(append.payloads).await {
+        let loglet_commit = match loglet.enqueue_batch(append.payloads.0).await {
```

Comment:
I would normally implement `From` and/or `Into` to avoid accessing the inner wrapped type here. This then can become `append.payloads.into()`. I also left a comment (you can find below) about reusing 

https://github.com/restatedev/restate/blob/a7884cbe045aaee48877aee5eb7b33dd46a18f82/crates/types/src/net/log_server.rs#L299

It has From/Into implementation (using `derive_more` crate)